static/app/views/automations/new-settings.tsx

```diff
@@ -61,6 +61,7 @@ function AutomationBreadcrumbs() {
 }
 
 const initialData = {
+  name: 'New Automation',
```

Comment:
Should we set this value on the initialData or just use as placeholder text? Setting this value would allow the form to submit with the name `New Automation`, but we'd prefer the users to give it a descriptive name.